lib/src/wrappers.dart

```diff
@@ -200,7 +200,7 @@ class DelegatingList<E> extends DelegatingIterable<E> implements List<E> {
   }
 
   @override
-  void fillRange(int start, int end, [E fillValue]) {
+  void fillRange(int start, int end, [E? fillValue]) {
```

Comment:
Did this signature come from the SDK? We should maybe file an issue, I don't know why we'd accept a nullable `fillValue` here. I guess because we don't have a default?

Reply:
Ya, https://github.com/dart-lang/sdk/blob/master/sdk_nnbd/lib/core/list.dart#L716.

I agree its weird, but yes I assume its because we couldn't make it a required parameter and not break other users. I wonder if we just used the `required` keyword though?

Reply:
I think `required` only works for named args. This unfortunate but I don't know if we have much choice. 😢